src/plugins/filter.ts

```
@@ -58,6 +381,121 @@ export async function apply(ctx: Context, config: Config) {
+                        Math.floor((now - info.lastUserMessageTime) / 1000)
+                    )
+                    triggerReason = `No new messages for ${elapsedSeconds}s`
+                }
+            }
+
+            if (!triggerReason) {
+                groupInfos[guildId] = info
+                continue
+            }
+
+            const triggered = await service.triggerCollect(session, triggerReason)
```

ESLint

[error] Replace `session,·triggerReason` with `⏎················session,⏎················triggerReason⏎············`

(prettier/prettier)